src/_guides/language/evolution.md

```diff
@@ -180,131 +182,157 @@ turn into edit-time analysis errors.
+### Dart 2.3
+_Released 8 May 2019_
+| [Dart 2.3 announcement](https://medium.com/dartlang/announcing-dart-2-3-optimized-for-building-user-interfaces-e84919ca1dff)
 
-[super parameters]: /language/constructors#super-parameters
+Dart 2.3 added three operators designed to improve code that performs
+list manipulation, such as declarative UI code.
 
-Dart 2.17 also removed some restrictions on named arguments.
-Named arguments can now be freely interleaved with positional arguments.
-The following code was invalid before 2.17,
-but is now allowed:
+The **[spread operator][]**
+enables unpacking the elements from one list into another.
+In the following example, the list returned by `buildMainElements()`
+is unpacked into the list being passed to the `children` argument:
 
 ```dart
-void main() {
-  test(skip: true, 'A test description', () {
-    // Very long function body here...
-  });
+Widget build(BuildContext context) {
+  return Column(children: [
+    Header(),
+    ...buildMainElements(),
+    Footer(),
+  ]);
 }
 ```
 
-To learn more about named parameters and arguments, see the
-[Named parameters][] section of the language tour.
+The **[collection if][]** operator enables adding elements that meet
+conditions.
+The following example adds a `FlatButton` element unless this element
+falls on the last page:
 
-[Named parameters]: /language/functions#named-parameters
+```dart
+Widget build(BuildContext context) {
+  return Column(children: [
+    Text(mainText),
+    if (page != pages.last)
+      FlatButton(child: Text('Next')),
+  ]);
+}
+```
 
-### Dart 2.18
+The **[collection for][]** operator enables building repeated elements.
+The following example adds one `HeadingAction` element for each section
+in `sections`:
 
-Dart 2.18 enhanced type inference. This change allows information flow between
-arguments in generic function calls. Before 2.18, if you didn't specify an
-argument's type in some methods, Dart returned errors. These type errors cited
-potential null occurrences. With 2.18, the compiler infers the argument type
-from other values in an invocation. You don't need to specify the argument type
-inline. To learn more, see [Type argument inference][].
+```dart
+Widget build(BuildContext context) {
+  return Column(children: [
+    Text(mainText),
+    for (var section in sections)
+      HeadingAction(section.heading),
+  ]);
+}
+```
 
-[Type argument inference]: /language/type-system#type-argument-inference
 
-Dart 2.18 also discontinued support for mixin classes that don't extend `Object`.
-To learn more about mixins,
-see [Adding features to a class: mixins][] in the language tour.
+### Dart 2.2
+_Released 26 February 2019_
+| [Dart 2.2 announcement](https://medium.com/dartlang/announcing-dart-2-2-faster-native-code-support-for-set-literals-7e2ab19cc86d)
 
-[Adding features to a class: mixins]: /language/mixins
+Dart supports literal lists and maps, but Dart 2.2 added support for
+**[set literals][]**:
```

Comment:
I think losing the "always supported" makes this a bit more confusing, since Dart 2.2 has already been out for a long time. If this was a release note, this would make more sense. Perhaps just focus on what was added?

```suggestion
Dart 2.2 added support for **[set literals][]**:
```